camayoc/tests/qpc/api/v1/sources/test_network_sources.py

```diff
@@ -213,3 +213,76 @@ def test_negative_update_invalid(
     assert_source_update_fails(original_data, src)
     src.hosts = ['1**2@33^']
     assert_source_update_fails(original_data, src)
+
+
+def test_create_empty_str_host_valid(shared_client, cleanup):
+    """Create a host manager source and then add host data with empty string.
+
+    :id: 4BFBF5B0-81BD-48A3-8C57-BAE55590285D
+    :description: Create network source of multiple hosts including an empty
+        string and credential.
+    :steps:
+        1) Create a network credential
+        2) Create a source with multiple host including empty string
+    :expectedresults: New source is created with all hosts except the empty
+        string filtered out.
+    """
+    EMPTY_STR_HOST_DATA = ['192.0.2.1', '192.0.2.10', '']
```

Comment:
This variable name should be in lower case since it is not a constant used in many places.